ubuntu-headless-qt/script/apt_install_base.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# ------------------------------------------------------------------------------------------#
+# This script is intended to be run inside a chroot environment to install essential packages
+# and perform system updates. It first checks if the script is executed as root, updates the
+# package list, and installs various required utilities and packages.
+# ------------------------------------------------------------------------------------------#
+
+export LC_ALL=C
+chmod 777 /tmp
+apt update
+apt clean
+apt autoclean
+apt upgrade -y
+apt update
+
+# Set DEBIAN_FRONTEND globally
+export DEBIAN_FRONTEND=noninteractive
+
+# Install debconf-utils for preconfiguring
+apt install -y debconf-utils
+
+# time zone data turn to default if not defined
+TIME_ZONE_AREA="${TIME_ZONE_AREA:=tzdata tzdata/Areas select Asia}"
+TIME_ZONE_CITY="${TIME_ZONE_CITY:=tzdata tzdata/Zones/Asia select Ho_Chi_Minh}"
+echo 'tzdata tzdata/Areas select Asia' | sudo debconf-set-selections
+echo 'tzdata tzdata/Zones/Asia select Ho_Chi_Minh' | sudo debconf-set-selections
```

Comment:
These two echo commands are hardcoded, correct?

Reply:
This is my mistake, I have fix that in the latest commit.

Reply:
```
echo 'tzdata tzdata/Areas select Asia' | sudo debconf-set-selections
echo 'tzdata tzdata/Zones/Asia select Ho_Chi_Minh' | sudo debconf-set-selections
```

Can you explain these commands? Do we need to set 'tzdata tzdata/Areas select Asia' and 'tzdata tzdata/Zones/Asia select Ho_Chi_Minh'? I think they are already set in config.ini

Reply:
> ```
> echo 'tzdata tzdata/Areas select Asia' | sudo debconf-set-selections
> echo 'tzdata tzdata/Zones/Asia select Ho_Chi_Minh' | sudo debconf-set-selections
> ```
> 
> Can you explain these commands? Do we need to set 'tzdata tzdata/Areas select Asia' and 'tzdata tzdata/Zones/Asia select Ho_Chi_Minh'? I think they are already set in config.ini

Those command are redundant. I have changed to use time zone parameters set in config.ini

Reply:
`tzdata tzdata/Zones/Asia`

Is this "Asia" a hardcode?

Reply:
> `tzdata tzdata/Zones/Asia`
> 
> Is this "Asia" a hardcode?

It's my mistake again. In next commit those settings will be changed to:
```c
echo "tzdata tzdata/Areas select $TIME_ZONE_AREA" | sudo debconf-set-selections
echo "tzdata tzdata/Zones/$TIME_ZONE_AREA select $TIME_ZONE_CITY" | sudo debconf-set-selections
```